### PR TITLE
[docs] Add an example page

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**

### CODE REVIEW

File: docs/articles/documentation/examples/README.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+This section lists examples from the [testcafe-examples](https://github.com/DevExpress/testcafe-examples) repository. All examples are ready to run.
+
+## Locate Elements
+
+Example | Description
+------- | ------------
+[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Find a header element with extra space characters by its trimmed text content.
+[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
+
+## Verifications
```
[the remark this answers is not in the document]
Comment (VasilyStrelyaev) [reply docs/articles/documentation/examples/README.md:17]: Excellent

File: docs/articles/documentation/examples/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+# Examples
+
+This section lists examples from the [testcafe-examples](https://github.com/DevExpress/testcafe-examples) repository. All examples are ready to run.
+
+## Locating Elements
+
+Example | Description
+------- | ------------
+[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Find a header element with extra space characters by trimmed text content.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:14]: Can you please clarify the functionality of this feature - for my understanding?
Does this feature search for elements that have additional space characters within the website/web page that is being tested?

File: docs/articles/documentation/examples/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:15]: Can you clarify what you mean by "an input with an XPath selector"? Does this feature search for XPath expressions within markup? If not, please clarify.

File: docs/articles/documentation/examples/README.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+## Testing the DOM
+
+Example | Description
+------- | ------------
+[Access Element Properties](https://github.com/DevExpress/testcafe-examples/tree/master/examples/element-properties) | Access DOM element's properties and verify them with assertions.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:21]:
```suggestion
[Access Element Properties](https://github.com/DevExpress/testcafe-examples/tree/master/examples/element-properties) | Access DOM element properties and verify them with assertions.
```

File: docs/articles/documentation/examples/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+[Test \<select\> Elements](https://github.com/DevExpress/testcafe-examples/tree/master/examples/test-select-elements) | Open a `<select>` element's drop-down list, click an item, and verify the value.
+[Page Reload](https://github.com/DevExpress/testcafe-examples/tree/master/examples/page-manipulation) | Execute custom code on the page to obtain data or manipulate the page state.
+
+## Test Organization
+
+Example | Description
+------- | ------------
+[Extract Code to Helpers](https://github.com/DevExpress/testcafe-examples/tree/master/examples/extract-code-to-helpers) | Extract test code to functions defined in a separate module.
+[Create Data-Driven Tests](https://github.com/DevExpress/testcafe-examples/tree/master/examples/create-data-driven-tests) | Create a test that performs parameterized actions.
+[Use Page Model](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-page-model) | Abstract out element selectors and common operations with these elements into a page model.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:32]:
```suggestion
[Use Page Model](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-page-model) | Extract element selectors and common operations with these elements to a page model.
```

File: docs/articles/documentation/examples/README.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+Example | Description
+------- | ------------
+[Inject Custom Client Scripts](https://github.com/DevExpress/testcafe-examples/tree/master/examples/client-scripts) | Inject Node.js modules into the tested webpage.
+[Import Third-Party Modules](https://github.com/DevExpress/testcafe-examples/tree/master/examples/import-third-party-modules) | Import a third-party Node.js module into test code.
+
+## Browser API Mocks
+
+Example | Description
+------- | ------------
+[Mock Date](https://github.com/DevExpress/testcafe-examples/tree/master/examples/mock-date) | Use [mockdate](https://www.npmjs.com/package/mockdate) to substitute the current data.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:45]:
```suggestion
[Mock Date](https://github.com/DevExpress/testcafe-examples/tree/master/examples/mock-date) | Substitute current data with [mockdate](https://www.npmjs.com/package/mockdate).
```

File: docs/articles/documentation/examples/README.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+[Mock Geolocation API](https://github.com/DevExpress/testcafe-examples/tree/master/examples/mock-geolocation-api) | Substitute the location in test code.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:46]: If you say "Substitute" in this sentence, then we also need to clarify what is being substituted.
Do you mean:

"Substitutes the current location with a mock location."

File: docs/articles/documentation/examples/README.md [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+
+## Wait Mechanisms
+
+Example | Description
+------- | ------------
+[Wait Until an Element Property Has a Specific Value](https://github.com/DevExpress/testcafe-examples/tree/master/examples/wait-for-element-property-value) | Use an assertion to pause the test until an element's width is `100%`.
+[Wait For File Download](https://github.com/DevExpress/testcafe-examples/tree/master/examples/wait-for-file-download) | Specify that a test should to wait for the file to download.
```
Comment (albertov05) [inline docs/articles/documentation/examples/README.md:53]:
```suggestion
[Wait For File Download](https://github.com/DevExpress/testcafe-examples/tree/master/examples/wait-for-file-download) | Specify that the test should wait for the file to download.
```

File: docs/articles/documentation/examples/README.md (hunk 2 above)
Comment (VasilyStrelyaev) [reply docs/articles/documentation/examples/README.md:14]: No, in this example, the page header consists of a meaningful part and some random number of space characters around it. We know the meaningful part (i.e. the "trimmed text"), so this example shows how we can find the header element using this part only, without knowing anything about these random space characters.

So, we can say something like:

*Shows how to ignore white-space characters when searching for an element by text.*
*Finds an element by text without regard to white-space characters at its ends.*

File: docs/articles/documentation/examples/README.md (hunk 3 above)
Comment (VasilyStrelyaev) [reply docs/articles/documentation/examples/README.md:15]: It means the example uses an XPath selector to find an input (I just didn't want to repeat the first column).

File: docs/articles/documentation/examples/README.md (hunk 2 above)
Comment (albertov05) [reply docs/articles/documentation/examples/README.md:14]:
```suggestion
[Find Element By Trimmed Text](https://github.com/DevExpress/testcafe-examples/tree/master/examples/find-element-by-trimmed-text) | Ignores white-space characters when searching for an element by text.
```

File: docs/articles/documentation/examples/README.md (hunk 3 above)
Comment (albertov05) [reply docs/articles/documentation/examples/README.md:15]:
```suggestion
[Use XPath Selectors](https://github.com/DevExpress/testcafe-examples/tree/master/examples/use-xpath-selectors) | Find an input with an XPath selector.
```

Comment (albertov05) [reply docs/articles/documentation/examples/README.md:15]: Thank you. Okay, your version is fine then.

**Pull request merged**